src/modules/Popup/Popup.js

```diff
@@ -228,7 +228,7 @@ export default class Popup extends Component {
 
     // Lets detect if the popup is out of the viewport and adjust
     // the position accordingly
-    const positions = _.without(POSITIONS, position)
+    const positions = _.without(POSITIONS, position).push(position)
```

Comment:
I add it to the end of the array, so that if no suitable position can be found, the selected (last) one is used. Otherwise it would just use the last one in POSITIONS, which is `bottom center`.